app/styles/app.scss

```diff
@@ -98,9 +97,9 @@ button {
   > * {
     pointer-events: none;
   }
+}
 
-  // Hide times in fastboot
-  .hide-in-fastboot.is-fastboot {
-    opacity: 0;
-  }
+// Hide times in fastboot
+.hide-in-fastboot.is-fastboot {
+  opacity: 0!important;
```

Comment:
What are your thoughts on when to use `!important`? In general I try to avoid it if possible. I can see this being an appropriate use, though.

Reply:
Oh whoops, didn't even mean to check this in with important, good catch.

I agree with you though, philosophically. It's usually a signal that css is messed, or that we're overriding something. But this isn't overriding anything—this was just an artifact from trying the first thing to make it work, realizing the actual problem, and not removing the first attempt. Correction coming.